diff --git a/hatchet_sdk/clients/rest/models/workflow_runs_metrics.py b/hatchet_sdk/clients/rest/models/workflow_runs_metrics.py
index 71b6351..5f70c44 100644
--- a/hatchet_sdk/clients/rest/models/workflow_runs_metrics.py
+++ b/hatchet_sdk/clients/rest/models/workflow_runs_metrics.py
@@ -22,13 +22,17 @@ from typing import Any, ClassVar, Dict, List, Optional, Set
 from pydantic import BaseModel, ConfigDict
 from typing_extensions import Self

+from hatchet_sdk.clients.rest.models.workflow_runs_metrics_counts import (
+    WorkflowRunsMetricsCounts,
+)
+

 class WorkflowRunsMetrics(BaseModel):
     """
     WorkflowRunsMetrics
     """  # noqa: E501

-    counts: Optional[Dict[str, Any]] = None
+    counts: Optional[WorkflowRunsMetricsCounts] = None
     __properties: ClassVar[List[str]] = ["counts"]

     model_config = ConfigDict(
